src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/GoalService.kt

```diff
@@ -190,55 +190,87 @@ class GoalService(
 
   @Transactional
   fun updateGoalStatus(goalUuid: UUID, updatedGoal: Goal): GoalEntity {
```

Comment:
I think we no longer need this method - it should be totally replaced by the three new functions